.tools/readmes/scanner.py

```diff
@@ -45,8 +46,16 @@ def _load_examples(self):
             f"{self.svc_name}_metadata.yaml", self.example_meta
         )
 
-    def set_example(self, language, service):
+    def _contains_language_version(self, example):
+        if self.lang_name in example["languages"]:
+            for version in example["languages"][self.lang_name]["versions"]:
+                if version["sdk_version"] == self.sdk_ver:
+                    return True
+        return False
```

Comment:
(opt)

        return self.sdk_ver in {
            version
            for version in example.get("languages", {})
            .get(self.lang_name, {})
            .get("versions", [])
        }

Reply:
I see what you're after, but I find your version a little harder to read.

Reply:
I find _your_ version harder to read